docs/v3/triggering.mdx

```diff
@@ -165,6 +165,43 @@ export const myTask = task({
 
 This is where it gets interesting. You can trigger a task and then wait for the result. This is useful when you need to call a different task and then use the result to continue with your task.
 
+<Accordion title="Don't use this in parallel, e.g. with `Promise.all()`">
+  Instead, use `batchTriggerAndWait()` if you can, or a for loop if you can't.
+
+<CodeGroup>
+```ts /trigger/batch.ts
+export const batchTask = task({
+  id: "batch-task",
+  run: async (payload: string) => {
+    const results = await childTask.batchTriggerAndWait({
+      items: [{ payload: "item1" }, { payload: "item2" }],
+      // optional: how many to run in parallel
+      batchOptions: { maxConcurrency: 1 },
```

Comment:
I didn't know we had this! Does it work?

Reply:
We do have this, but it's a mystery (I didn't actually check)

Reply:
@ericallam does this work?